src/mempool.cpp

```diff
@@ -142,20 +151,51 @@ void MemoryPool::deallocate(void* ptr, size_t size) {
     }
 }
 
+void create_mempool(uv_work_t *req) {
```

Comment:
can we move these helper function into infinistore.cpp, because it depends on libuv?